### ISSUE TITLE
cmake Error on Ubuntu 18-04

### ISSUE THREAD

**Issue by goetzmoritz** [colmap/colmap#421]
I can't cmake on Ubuntu 18:04. Any ideas? Somehow cmake can't read the CUDA version. CUDA is enabled however.

```
CC=/usr/bin/gcc-6 CXX=/usr/bin/g++-6 cmake ..
-- The C compiler identification is GNU 6.4.0
-- The CXX compiler identification is GNU 6.4.0
-- Check for working C compiler: /usr/bin/gcc-6
-- Check for working C compiler: /usr/bin/gcc-6 -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/g++-6
-- Check for working CXX compiler: /usr/bin/g++-6 -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found installed version of Eigen: /usr/lib/cmake/eigen3
-- Found required Ceres dependency: Eigen version 3.3.4 in /usr/include/eigen3
-- Found required Ceres dependency: glog
-- Found installed version of gflags: /usr/lib/x86_64-linux-gnu/cmake/gflags
-- Detected gflags version: 2.2.1
-- Found required Ceres dependency: gflags
-- Found Ceres version: 2.0.0 installed in: /usr/local with components: [EigenSparse, SparseLinearAlgebraLibrary, LAPACK, SuiteSparse, CXSparse, SchurSpecializations]
-- Boost version: 1.65.1
-- Found the following Boost libraries:
--   program_options
--   filesystem
--   graph
--   regex
--   system
--   unit_test_framework
-- Found Eigen3: /usr/include/eigen3 (Required is at least version "2.91.0") 
-- Found Eigen
--   Includes : /usr/include/eigen3
-- Found FreeImage
--   Includes : /usr/include
--   Libraries : /usr/lib/x86_64-linux-gnu/libfreeimage.so
-- Found Glog
--   Includes : /usr/include
--   Libraries : /usr/lib/x86_64-linux-gnu/libglog.so
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
-- Found Glew
--   Includes : /usr/include
--   Libraries : /usr/lib/x86_64-linux-gnu/libGLEW.so
-- Found Git: /usr/bin/git (found version "2.17.1") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Looking for pthread_create
-- Looking for pthread_create - not found
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found Qt
--   Module : /usr/lib/x86_64-linux-gnu/cmake/Qt5Core
--   Module : /usr/lib/x86_64-linux-gnu/cmake/Qt5OpenGL
--   Module : /usr/lib/x86_64-linux-gnu/cmake/Qt5Widgets
-- Build type not specified, using Release
-- Enabling SIMD support
-- Enabling OpenMP support
-- Disabling interprocedural optimization
CMake Error at cmake/SelectCudaComputeArch.cmake:235 (message):
  Unknown CUDA Architecture Name OFF in CUDA_SELECT_NVCC_ARCH_FLAGS
Call Stack (most recent call first):
  CMakeLists.txt:186 (CUDA_SELECT_NVCC_ARCH_FLAGS)


CMake Error at cmake/SelectCudaComputeArch.cmake:239 (message):
  arch_bin wasn't set for some reason
Call Stack (most recent call first):
  CMakeLists.txt:186 (CUDA_SELECT_NVCC_ARCH_FLAGS)


-- Enabling CUDA support (version: 9.1, archs: )
-- Enabling OpenGL support
-- Disabling profiling support
-- Disabling CGAL support
-- Configuring incomplete, errors occurred!
See also "/home/render/temp/colmap/build/CMakeFiles/CMakeOutput.log".
See also "/home/render/temp/colmap/build/CMakeFiles/CMakeError.log".
```

**Comment by goetzmoritz** [comment]
works fine. Thanks!

**Comment by philleer** [comment]
@goetzmoritz how did you fix it? i met the same issue...

**Comment by ahojnnes** [comment]
Which CMake Version do you use?

**Comment by philleer** [comment]
@ahojnnes thank you, i have fixed it according to your suggestion from #594 
my cmake version is 3.5.1 and now it works fine.
Much thx!
